informer/informer.go

```diff
@@ -82,7 +82,7 @@ func New(config Config) (*Informer, error) {
 	}
 
 	newInformer := &Informer{
-		// Settings.
+		// Dependencies.
```

Comment:
We agreed to remove those comments, but whatever.

Reply:
You are right. I just found it kind of useful for the open source projects, but yeah.